src/node_options.cc

```diff
@@ -435,6 +435,10 @@ EnvironmentOptionsParser::EnvironmentOptionsParser() {
             "specified sampling interval in bytes for the V8 heap "
             "profile generated with --heap-prof. (default: 512 * 1024)",
             &EnvironmentOptions::heap_prof_interval);
+  AddOption("--max-http-header-size",
+            "set the maximum size of HTTP headers (default: 8KB)",
```

Comment:
Perhaps: (to be consistent with docs)
```suggestion
            "set the maximum size of HTTP headers (default: 8192 (8KB))",
```

Reply:
Done